src/main/java/frc/robot/commands/GoToPose.java

```diff
@@ -0,0 +1,67 @@
+package frc.robot.commands;
+
+import java.util.function.Supplier;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.constants.TestConstants;
+import frc.robot.subsystems.Drivetrain;
+
+/**
+ * Tests the odometry of the robot by driving a certain distance and calculating the error.
+ */
+public class GoToPose extends CommandBase {
+
```

Comment:
This command seems to be the same as the odometery test command. So i would either remove the odometery test command or this one.
The only difference in the commands is that one takes a Transform2d, a distance to move relative to the robot, and this one takes an absolute field pose. They can be made the same command by just adding another constructor that takes a robot relative Transform2d